src/views/Simulation/utils/MinimapContainer.js

```diff
@@ -40,6 +40,20 @@ export class MinimapContainer extends Container {
     this.minimapAppRef.current.stage.on('wheel', this.onWheel);
   }
 
+  createNodeTextures() {
+    const createNodeTexture = (fillColor) => {
+      const nodeGraphics = () => new GraphicsContext().rect(0, 0, 2, 2).fill(fillColor);
```

Comment:
The variable name could be improved, this is not a `Graphics` but a `GraphicsContext` :slightly_smiling_face: 
Also, do we need a function here? It's only called once below, maybe we should store the graphics context in a constant directly, without having a function to call.